metaflow/plugins/project_decorator.py

```diff
@@ -13,6 +13,19 @@
 
 
 class ProjectDecorator(FlowDecorator):
+    """
+    Specifies what projects belong to the same project.
```

Comment:
you mean, flows? Or just "Specifies the project for this flow"

Reply:
flows, right, fixed